stdlib/public/Glibc/Glibc.swift

```diff
@@ -29,6 +29,100 @@ public var errno: Int32 {
 // fcntl.h
 //===----------------------------------------------------------------------===//
 
+@warn_unused_result
+@_silgen_name("_swift_Glibc_open")
+func _swift_Glibc_open(path: UnsafePointer<CChar>,
+  _ oflag: CInt,
+  _ mode: mode_t
+) -> CInt
+
+@warn_unused_result
+@_silgen_name("_swift_Glibc_openat")
+func _swift_Glibc_openat(
+  fd: CInt,
+  _ path: UnsafePointer<CChar>,
+  _ oflag: CInt,
+  _ mode: mode_t
+) -> CInt
+
+@warn_unused_result
+public func open(
+  path: UnsafePointer<CChar>,
+  _ oflag: CInt
+) -> CInt {
```

Comment:
Because that's how the man page describes the function: there are three variants, one accepts void, another accepts a pointer, and another accepts int.  It does not describe the void one as a variant of the int one.